### PR TITLE
Add in ResNet + EfficientNet TF2 classifier tests

### PR DESCRIPTION
**Pull request by allenwang28**

### CODE REVIEW

**Comment by allenwang28** [comment]
Tested:

Screen Shot 2020-03-30 at 12 15 50 PM

File: templates/garden/nightly/classifier-efficientnet.libsonnet [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+      },
+      eval_dataset+: {
+        batch_size: 1024,
+      },
+    },
+  },
+
+  configs: [
+    efficientnet + v2_8 + functional,
+    efficientnet + v3_8 + functional, efficientnet + v2_8 + convergence,
```
Comment (will-cromar) [inline templates/garden/nightly/classifier-efficientnet.libsonnet]: Nit: add a line break here

Comment (allenwang28) [reply templates/garden/nightly/classifier-efficientnet.libsonnet]: Done

**Comment by allenwang28** [comment]
Thanks, I added in the regression tests and then generated the CronJobs. PTAL!

File: templates/garden/nightly/classifier-efficientnet.libsonnet [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+  },
+  local convergence = mixins.Convergence {
+    paramsOverride+: {
+      train+: {
+        epochs: 350, 
+      },
+    },
+    regressionTestConfig+: {
+      metric_success_conditions+: {
+        "Accuracy/test_final": {
```
Comment (zcain117) [inline templates/garden/nightly/classifier-efficientnet.libsonnet]: This should match the name of your accuracy tag in tensorboard + "_final"
 
The PyTorch resnet50 template gets that string from [here](https://github.com/pytorch/xla/blob/master/test/test_train_imagenet.py#L238)

Comment (allenwang28) [reply templates/garden/nightly/classifier-efficientnet.libsonnet]: That makes sense, thanks! Changed it to 'epoch_accuracy_final' as that's what Keras uses for accuracy.

File: templates/garden/nightly/classifier-efficientnet.libsonnet [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+  },
+  local convergence = mixins.Convergence {
+    paramsOverride+: {
+      train+: {
+        epochs: 350, 
+      },
+    },
+    regressionTestConfig+: {
+      metric_success_conditions+: {
+        "epoch_accuracy_final": {
```
Comment (will-cromar) [inline templates/garden/nightly/classifier-efficientnet.libsonnet]: micro-nit:  this doesn't have to be quoted since it's a valid name in JSonnet

Comment (allenwang28) [reply templates/garden/nightly/classifier-efficientnet.libsonnet]: Thanks, done
